bootstrapgenerator/src/main/java/nl/openweb/hippo/groovy/model/Constants.java

```diff
@@ -19,44 +19,63 @@
 /**
  * Constants for generating bootstrap xml
  */
-public interface Constants {
+public final class Constants {
+    private Constants(){
+        //no public construction
+    }
+
+    public final class Files {
+        private Files(){
+            //no public construction
+        }
 
-    interface Files {
-        String GROOVY_EXTENSION = ".groovy";
-        String XML_EXTENSION = ".xml";
-        String YAML_EXTENSION = ".yaml";
-        String ECM_EXTENSIONS_NAME = "hippoecm-extension.xml";
+        public static final String GROOVY_EXTENSION = ".groovy";
+        public static final String XML_EXTENSION = ".xml";
+        public static final String YAML_EXTENSION = ".yaml";
+        public static final String ECM_EXTENSIONS_NAME = "hippoecm-extension.xml";
     }
 
-    interface PropertyName {
-        String JCR_PRIMARY_TYPE = "jcr:primaryType";
-        String HIPPOSYS_BATCHSIZE = "hipposys:batchsize";
-        String HIPPOSYS_DRYRUN = "hipposys:dryrun";
-        String HIPPOSYS_PARAMETERS = "hipposys:parameters";
-        String HIPPOSYS_QUERY = "hipposys:query";
-        String HIPPOSYS_SCRIPT = "hipposys:script";
-        String HIPPOSYS_THROTTLE = "hipposys:throttle";
-        String HIPPOSYS_DESCRIPTION = "hipposys:description";
-        String HIPPOSYS_PATH = "hipposys:path";
-        String HIPPO_SEQUENCE = "hippo:sequence";
-        String HIPPO_CONTENTRESOURCE = "hippo:contentresource";
-        String HIPPO_CONTENTROOT = "hippo:contentroot";
-        String HIPPO_RELOADONSTARTUP = "hippo:reloadonstartup";
-        String HIPPO_VERSION = "hippo:version";
+    public final class PropertyName {
```

Comment:
not an interface like the others inside this Constants.class

Reply:
I don't understand the comment

Reply:
It's ok this way.